ts-fold.el

```diff
@@ -446,6 +451,71 @@ more information."
          (end (tsc-node-start-position end-node)))
     (ts-fold--cons-add (cons beg end) offset)))
 
+;;+ OCaml
+
+(defun ts-fold-range-ocaml-comment (node offset)
+  "Define fold range for `comment'.
+
+For arguments NODE and OFFSET, see function `ts-fold-range-seq' for
+more information."
+  (unless (ts-fold--one-liner-node node)
+    (when-let*
+        ((text  (tsc-node-text node))
+         (beg   (if (string-prefix-p "(* " text)
+                    (+ 2 (tsc-node-start-position node))
+                  (+ 3 (tsc-node-start-position node))))
+         (end   (- (tsc-node-end-position node) 2)))
+      (cons beg end))))
```

Comment:
To address warning `unused variable 'offset'`, we should use:

```
(ts-fold--cons-add (cons beg end) offset))
```

instead of,

```
(cons beg end)
```

Reply:
Oh yes, I'll change that